### DISCUSSION TITLE
Questions on synthtiger configuration to generate a training dataset

### DISCUSSION THREAD

**Discussion by chmaz** [mindee/doctr#1790] [Q&A]
Hello,

I am trying to use synthtiger to generate training data for my use case but feels a bit lost and have some questions:
(1) is it possible to configure the config_horizontal.yaml file so that the background is white uniform (before applying the noise filters), or possibly of a shade of gray configurable between white and lightgray?
(2) similarly, is it possible to opt out from truncated characters added in the background on the top or bottom margins of the generated images?

Thanks in advance for you help,

Best,

Chris

**Comment by felixdittrich92** [comment]
Hey @chmaz :wave:,

I think that's more questions for the SynthTiger repo :sweat_smile: 

If i remember setting `color` `colorize` to 0 generates white / gray backgrounds

I have modified SynthTiger a bit (it's nothing official yet was only for some own experiments) but maybe it helps: 

https://github.com/felixdittrich92/synthtiger/tree/doctr-modified

branch: `doctr-modified` 

I used this to generate a multilingual dataset which was used to fine tune: https://huggingface.co/Felix92/doctr-torch-parseq-multilingual-v1

Best,

Felix

**Reply by chmaz** [reply]
Hi @felixdittrich92,

I posted the questions here because I am using your fork of the ClovaAI repo and did not find where to ask questions there ;)
Thanks for the answer! Indeed setting color colorize to 0 generates white / gray backgrounds. What I am trying to achieve is to generate only white backgrounds with possibly the color of the text being a shade or dark gray or simply black.  I have tried to look at the ClovaAI documentation but was not successful.

Best,

Chris

**Reply by felixdittrich92** [reply]
Hi @chmaz :wave:,

I think the easiest would be to modify the following to your own needs:

https://github.com/felixdittrich92/synthtiger/blob/15b13f5a96c70e9bf7c049b2d04f679446dfbcda/examples/synthtiger/template.py#L115

additional you can replace the images with a simple white one and remove the others:
https://github.com/felixdittrich92/synthtiger/tree/doctr-modified/resources/image

Best,
Felix :)

**Comment by chmaz** [comment]
Thanks a lot!  I will try
Best,

Chris
